components/brave_wallet/browser/fil_transaction.h

```diff
@@ -51,10 +50,10 @@ class FilTransaction {
   }
   void set_gas_limit(int64_t gas_limit) { gas_limit_ = gas_limit; }
   void set_max_fee(const std::string& max_fee) { max_fee_ = max_fee; }
-  void set_cid(const std::string& cid) { cid_ = cid; }
 
   std::string GetMessageToSign() const;
   base::Value ToValue() const;
+  mojom::FilTxDataPtr ToFilTxData() const;
```

Comment:
is this covered by fil transaction unittest?

Reply:
it is covered by FilTxMeta but added FilTransaction too